project-apps-dev/ProgrammersOnly/mongodb_service.tf

```diff
@@ -44,7 +44,8 @@ resource "aws_ecr_repository_policy" "mongodb" {
                 "ecr:BatchDeleteImage",
                 "ecr:SetRepositoryPolicy",
                 "ecr:DeleteRepositoryPolicy"
-            ]
+            ],
+            "Resource": "*"
```

Comment:
This is not needed
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository_policy

Reply:
Done.